styles.css

```diff
@@ -35,4 +35,24 @@ p {
 
 a {
     color: #3498db;
+}
+.button{
+    
+    color: red;
+}
+.btn{
+    position: relative;
+    left: -1px;
+    top: -1px;
+    padding: 7px 16px;
+    color: red;
+    border-radius: 30px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-size: medium;
+    border:none;
+}
+
+.btn:hover{
+    color:white;
+    background-color: Red;
```

Comment:
You could work on making code prettier. Your future teammates will appreciate it. Take a look at how other code here is structured.

Also, I think this is not needed:
```
position: relative;
left: -1px;
top: -1px;
```

Reply:
> You could work on making code prettier. Your future teammates will appreciate it. Take a look at how other code here is structured.
> 
> Also, I think this is not needed:
> 
> ```
> position: relative;
> left: -1px;
> top: -1px;
> ```
> 
> Try not to use minuses if not needed.

cool